spec/puppetserver/ca/list_action_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'puppetserver/ca/list_action'
+require 'puppetserver/ca/cli'
+require 'puppetserver/ca/logger'
+
+RSpec.describe 'Puppetserver::Ca::ListAction' do
+  let(:err)    { StringIO.new }
+  let(:out)    { StringIO.new }
+  let(:logger) { Puppetserver::Ca::Logger.new(:info, out, err) }
+  let(:action) { Puppetserver::Ca::ListAction.new(logger) }
+  let(:result) {[{"name"=>"foo", "state"=>"signed", "dns_alt_names"=>["DNS:foo", "DNS:bar"],
```

Comment:
Oh gross, even the API calls them `dns_alt_names`?

Reply:
yes :( but we list them as ``alt names`` when we print them out